trelabels.js

```diff
@@ -68,13 +68,13 @@ function addTrelabelsButton() {
     return;
   }
 
-  if ($('.header-boards-button').length === 0) {
```

Comment:
`.header-boards-button` still exists.

![image](https://user-images.githubusercontent.com/954168/57438929-3b9d8680-7245-11e9-94b1-097b8b3ef548.png)

Reply:
This is really odd, in my browser I have this:

![Anmerkung 2019-05-09 105419](https://user-images.githubusercontent.com/2011264/57440831-03984280-7249-11e9-8f25-87cd78801e4a.jpg)

Therefore I chose a different approach in that line.

Reply:
Can you check with the Chrome Console if `$('#header').find('input').parent()` gives you the same element as `input[type="search"]` ?

The input in the "old" version of Trello is a `type="text"`...

Reply:
Yep, that works.

Reply:
I can blindly push the changes and hope for the best, but from what I see on your screenshot, all class names are now generated. So the button and menu, which uses the old class names, will still not show up correctly...

Reply:
I will do another test round and try to test it in both variants.

Reply:
`$('#header').find('input').parent().prepend(button);` looks good in Chrome ("new") and Firefox ("old"). The Trelabel button is there and functional. However, I don't have any other extension for Trello, so I hope this will not interfere.